### PR TITLE
[TACHYON-648] Add unit test of EvictorFactory

### PR DESCRIPTION
**Pull request by luoli523**
[TACHYON-648] Add unit test of EvictorFactory
https://tachyon.atlassian.net/browse/TACHYON-648

### CODE REVIEW

File: servers/src/test/java/tachyon/worker/block/evictor/EvictorFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  private BlockMetadataManagerView mManagerView;
+  
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+
+  @Before
+  public void before() throws IOException {
+    File tempFolder = mTestFolder.newFolder();
+    mMetaManager = EvictorTestUtils.defaultMetadataManager(tempFolder.getAbsolutePath());
+    mManagerView = new BlockMetadataManagerView(mMetaManager, Collections.<Integer>emptySet(),
```
Comment (gpang) [inline servers/src/test/java/tachyon/worker/block/evictor/EvictorFactoryTest.java]: Does a new BlockMetadataManagerView need to be created before every test, or can a single one be shared for all the tests?

File: servers/src/test/java/tachyon/worker/block/evictor/EvictorFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import tachyon.worker.block.BlockMetadataManager;
+import tachyon.worker.block.BlockMetadataManagerView;
+
+/*
+ * Test EvictorFactory by passing different AllocatorType 
+ * and test if it generate the correct Evictor instance.
+ */
+public class EvictorFactoryTest {
+  private BlockMetadataManager mMetaManager;
```
Comment (gpang) [inline servers/src/test/java/tachyon/worker/block/evictor/EvictorFactoryTest.java]: It doesn't look like 'mMetaManager' is used anywhere in the tests, but only used to create the view.

Comment (luoli523) [inline servers/src/test/java/tachyon/worker/block/evictor/EvictorFactoryTest.java]: But seems like it is necessary that we have a BlockMetadataManager instance to create a view and then we can create Evictor instance?

What about the EvictorTestUtils holds a static BlockMetadataManager instance and a static BlockMetadataManagerView instance and we use them anywhere?

Comment (apc999) [inline servers/src/test/java/tachyon/worker/block/evictor/EvictorFactoryTest.java]: A simpler solution might be just making `mMetaManager` and `mMetaManagerView` static member to share among tests in this `EvictorFactoryTest`? Changing `EvictorTestUtils` might be too much for this PR :)

**Comment by apc999** [comment]
@apc999 please review this PR

**Pull request closed without merging**